rskj-core/src/test/java/co/rsk/peg/FederationTestUtils.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import org.ethereum.crypto.ECKey;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FederationTestUtils {
+    public static List<FederationMember> getFederationMembers(int amount, int offset) {
```

Comment:
I think amount is used for money, maybe use a name like numberOfFeds or memberCount. 
Also offset is called always with value 1, i think removing this parameter will be clearer for the tests, or make it optional

Reply:
:+1: